java-extensions/jni-connector/src/main/java/com/starrocks/jni/connector/OffHeapColumnVector.java

```diff
@@ -696,4 +723,34 @@ public byte[] changeByteOrder(byte[] bytes) {
         }
         return bytes;
     }
+
+    public static int convertToDate(int year, int month, int day) {
```

Comment:
Document the location of these logical components within the CPP file in the comments.